kloppy/tests/test_statsbomb.py

```diff
@@ -863,6 +864,10 @@ def test_loose_ground_duel_qualfiers(self, dataset: EventDataset):
             DuelType.GROUND,
         ]
 
+    def test_counter_attack_qualifier(self, dataset: EventDataset):
```

Comment:
I think it would be good if we added a few more tests. For instance, check the number of counter-attacking events and check whether a non-counter-attacking event doesn't have the qualifier.